### ISSUE TITLE
FR: Allow fireballs.tbl to have particle_effects entries (or something with similar functionality) instead of bitmaps.

### ISSUE THREAD

**Issue by MoerasGrizzly** [scp-fs2open/fs2open.github.com#5942]
Would allow people to define fancier, generic explosion effects without having to go through several different table entries.

I suppose such entries would have to make use of +Parent Size Factor: rather then +Size, given how fireballs scale.

**Comment by Baezon** [comment]
For what it's worth, particle effects can replace death fireballs by using $Ship Death Effect and removing the fireball.

The point about such usages being able to use +Parent Size Factor is a really good one, though.

**Comment by MoerasGrizzly** [comment]
> For what it's worth, particle effects can replace death fireballs by using $Ship Death Effect and removing the fireball.

Yes, and I do use that feature quite a bit, but fireball.tbl is used by so many different functions that being able to shove some "one size fits all" effects in there can be very useful.
